pkg/run/trace.go

```diff
@@ -235,7 +241,7 @@ func (tb *runTree) constructSpan(ctx context.Context, s *cqrs.Span) (*rpbv2.RunS
 		SpanId:       s.SpanID,
 		Name:         name,
 		Status:       rpbv2.SpanStatus_RUNNING,
-		QueuedAt:     timestamppb.New(ulid.Time(runID.Time())),
+		QueuedAt:     timestamppb.New(ulid.Time(runID.Time())), // TODO: need to be different values for steps
```

Comment:
Do we have any other timestamp for this? A change in the future may be to allow run IDs to be created _before_ being scheduled, which might skew the `QueuedAt` time.

Reply:
This is fine for function spans but not the execution spans, which is what the comment is referring to.
I'm gonna annotate the queue item's timestamp for queued at since that's what it is for executions.

Reply:
seems like we don't have that information atm, so I'm just going to calculate the queue time from the delay.